packages/client/src/components/SafeSettings/components/EditThreshold/ReviewSafeEdits.js

```diff
@@ -1,58 +1,66 @@
+    if (newOwner) {
+      return "Add Owner"
+    }
+    else if (ownerToBeRemoved) {
+      return "Remove Owner"
+    }
+    else {
+      return "Signature Threshold"
+    }
+  }
+  const renderMainSection = () => {
+    if (!owner) {
+      return (
+        <>
+          <h1 className="is-family-monospace has-text-black has-text-weight-bold">{`${newThreshold} of ${allSafeOwners.length} owners`}</h1>
+        </>
+      )
+    }
+    if (owner.name)
+      return (
+        <>
+          <h1 className="is-family-monospace has-text-black has-text-weight-bold">{owner.name}</h1>
+          <div className="has-text-black has-text-weight-bold">{owner.address}</div>
+        </>
+      );
+    return (
+      <h1 className="has-text-black has-text-weight-bold">{owner.address}</h1>
+    );
+  }
 
   return (
     <>
-      <div className="p-5">
-        <h2 className="is-size-4 has-text-black">Review updates</h2>
-        <p className="has-text-grey">
-          Confirm the details below before submitting
-        </p>
+      <div className="p-4">
+        <h2 className="is-size-4 has-text-black">Review Updates</h2>
       </div>
-      <div className="border-light-top p-5 has-text-grey">
-        <div className="is-flex pt-1 pb-5">
-          <div className="is-flex flex-1 is-flex-direction-column border-light-right">
-            <label>Number of owners</label>
-            <div className="has-text-black is-size-5">
-              {allSafeOwners.length}
-            </div>
-          </div>
-          <div className="is-flex flex-1 is-flex-direction-column pl-5">
-            <label>Signature threshold</label>
-            <div className="is-flex is-align-items-center">
-              <span
-                className="is-size-5 has-text-black"
-                style={{ whiteSpace: "nowrap" }}
-              >
-                {newThreshold} of {allSafeOwners.length}
-              </span>
-              <div className="is-flex ml-1" style={{ height: 16 }}>
-                <SignatureBar
-                  threshold={newThreshold}
-                  safeOwners={allSafeOwners}
-                />
-              </div>
-            </div>
-          </div>
+      <div className="border-light-top p-4 has-text-grey">
+        <p className="has-text-grey">{mainSectionHeaderText()}</p>
+        <div className="pb-4">
+          {renderMainSection()}
         </div>
-        {newOwner && renderRow("New owner", newOwner.name)}
-        {newOwner && renderRow("Address", newOwner.address)}
-        {renderRow("Transaction date", new Date().toLocaleDateString("en-US"))}
-        <div className="is-flex is-align-items-center mt-5">
+        {renderRow("Proposed On", new Date().toLocaleDateString("en-US"))}
+        {/* only render signature threshold row if edit is AddOwner or RemoveOwner */}
+        {newOwner || ownerToBeRemoved ? renderRow("Signature Threshold", `${newThreshold} of ${allSafeOwners.length} owner(s)`) : <></>}
```

Comment:
```suggestion
        {newOwner || ownerToBeRemoved ? renderRow("Signature Threshold", `${newThreshold} of ${allSafeOwners.length} owner(s)`) : null}
```
not big deal but convention usually to return null vs empty fragment